### ISSUE TITLE
Some keyboard navigation updates have been lost

### ISSUE THREAD

**Issue by StephDriver** [openlibhums/janeway#5387]
this is looking at r-1.9.x. 

1. keyboard trap, cannot dismiss enlarged table modal.
- only affects Clean and Clarity. 
- Clean was working in PR #4901.  So compare with that to find what has been lost. 
- To reproduce. Navigate an article which has tables using the keyboard. Open a modal, try to dismiss the modal by tabbing to the close button - you can't!   Compare with Material and OLH where it works fine. 

2. on navigating an in page link, on tabbing the keyboard focus returns to the top of the page, rather than the landing element. 
- affects OLH, untested on other themes.
- was working when introduced in PR #4789

**Comment by mauromsl** [comment]
Planning: This issue is being resolved by restoring the behaviour from previously merged PRs

**Comment by ajrbyers** [comment]
The clarity fix is included here: https://github.com/openlibhums/clarity/pull/13
